libavcodec/vaapi_av1.c

```diff
@@ -241,6 +241,7 @@ static int vaapi_av1_start_frame(AVCodecContext *avctx,
             .disable_frame_end_update_cdf = frame_header->disable_frame_end_update_cdf,
             .uniform_tile_spacing_flag    = frame_header->uniform_tile_spacing_flag,
             .allow_warped_motion          = frame_header->allow_warped_motion,
+            .large_scale_tile             = frame_header->large_scale_tile,
```

Comment:
Please add a check for va-api version at compile time. large_scale_tile is not available in old versions of libva.

Reply:
Please ignore the above comment.